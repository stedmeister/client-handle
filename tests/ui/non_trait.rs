use client_handle::async_tokio_handle;

#[async_tokio_handle]
struct SomeStruct;

#[async_tokio_handle]
enum SomeEnum {}

fn main() {}