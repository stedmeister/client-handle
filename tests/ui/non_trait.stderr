error: The `async_tokio_handle` macro only works on traits
 --> tests/ui/non_trait.rs:4:1
  |
4 | struct SomeStruct;
  | ^^^^^^^^^^^^^^^^^^

error: The `async_tokio_handle` macro only works on traits
 --> tests/ui/non_trait.rs:7:1
  |
7 | enum SomeEnum {}
  | ^^^^^^^^^^^^^^^^
